docs/examples/rs-tutorial.rst

```diff
@@ -0,0 +1,163 @@
+
+.. code-block:: python
+
+    from keras.datasets import cifar10
+
+    class GetDataCifar():
+        def execute(self):
+            ((train_imgs, train_labels),
+            (test_imgs, test_labels)) = cifar10.load_data()
+            return train_imgs, train_labels, test_imgs, test_labels
+
+Train CIFAR-10
+~~~~~~~~~~~~~~
+.. figure:: train-basic.png
+    :align: center
+
+This pipeline gives a very basic example of how to create, train, and evaluate a simple CNN. The primary takeaway from this pipeline should be the overall structure of a training pipeline, which should follow the following steps in most cases:
+
+1. Load data
+2. Define the loss, optimizer, and other metrics
+3. Compile model, with loss, metrics, and optimizer, using the **compile()** method
+4. Train model using the **fit()** method, which requires the training inputs and outputs
+5. Output the trained model for serialization and/or utilization in subsequent nodes
+
+.. code-block:: python
+
+    import numpy as np
+    import keras
+
+    class TrainBasic():
+        def __init__(self, model, epochs=20, batch_size=32, shuffle=True):
+            self.model = model
+            self.epochs = epochs
+            self.batch_size = batch_size
+            self.shuffle = shuffle
+            return
+
+        def execute(self, train_imgs, train_labels):
+            opt = keras.optimizers.rmsprop(lr=0.001)
+            self.model.compile(loss='sparse_categorical_crossentropy',
+                            optimizer=opt,
+                            metrics=['sparse_categorical_accuracy'])
+            self.model.fit(train_imgs,
+                        train_labels,
+                        batch_size=self.batch_size,
+                        epochs=self.epochs,
+                        shuffle=self.shuffle,
+                        verbose=2)
+            model = self.model
+            return model
+
+.. code-block:: python
+
+    class EvalBasic():
+        def __init__(self):
+            return
+
+        def execute(self, model, test_imgs, test_labels):
+            results = model.evaluate(test_imgs, test_labels, verbose=0)
+            for i, metric in enumerate(model.metrics_names):
+                print(metric,'-',results[i])
+            return results
+
+Train-Test
+~~~~~~~~~~
+.. figure:: train-basic.png
+    :align: center
+
+This pipeline provides an example of how one might train and evaluate a redshift estimation model.For the training process, there are two primary additions that should be noted.
+
+First, the **Train** class has been given a function named **to_categorical**.  Because we are using categorization models for redshift estimation in this tutorial, the keras model expects the output labels to be either one-hot vectors or a single integer where the position/value indicates the range in which the true redshift value falls. This function converts the continuous redshift values into the necessary discrete, categorical format.
```

Comment:
Rather than make significant changes to this sentence, I instead referred back to our method being a simplified version of Pasquet et. al. This should make the jump into discussing classification models a bit less jarring.